NewTutorials/Tutorial/TutorialCasesViewController.swift

```diff
@@ -61,14 +62,23 @@ class PlayerViewControllerWrapper: UIViewController {
             view.setNeedsLayout()
         }
     }
-    let playerViewController = PlayerViewController()
+    
+    private let playerViewController = PlayerViewController()
     private let activityIndicatorView: UIActivityIndicatorView = {
         let activityIndicatorView = UIActivityIndicatorView()
         activityIndicatorView.activityIndicatorViewStyle = .gray
         activityIndicatorView.isHidden = true
         return activityIndicatorView
     }()
     
+    func attach(player: Player) {
+        playerViewController.player = player
+    }
+    
+    func attach(controls: ContentControlsViewController) {
```

Comment:
I don't see any customisation code - so what is the reason of having this method?